src/main/java/com/checkmarx/flow/config/ScmConfigOverrider.java

```diff
@@ -12,19 +12,19 @@
 @Slf4j
 public class ScmConfigOverrider {
 
-    public String determineConfigToken(RepoProperties properties, String scmInstance) {
+    String determineConfigToken(RepoProperties properties, String scmInstance) {
```

Comment:
Can we replace the "determine..." methods with a single method that returns a DTO? The DTO will contain the result of merging instance properties and default properties.

Reply:
Done